server/apps/publish/archive_publish.py

```diff
@@ -92,6 +92,21 @@ def on_update(self, updates, original):
         if validation_errors[0]:
             raise ValidationError(validation_errors)
 
+        # We do not allow packages to be published if any items in the package do not validate
+        if original[ITEM_TYPE] == CONTENT_TYPE.COMPOSITE:
+            items = [ref.get('residRef') for group in original.get('groups', [])
+                     for ref in group.get('refs', []) if 'residRef' in ref]
+            if items:
+                for guid in items:
+                    doc = super().find_one(req=None, _id=guid)
```

Comment:
It's already done in pr#901. https://github.com/superdesk/superdesk/blob/master/server/apps/publish/archive_publish.py#L89-L91